### PR TITLE
Added port specification to postgresDestination

### PR DESCRIPTION
**Pull request by suityou01**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

Cannot specify port for postgres destination

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A it should connect to different port

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/destinations/README.md [hunk 1]
```diff
@@ -65,7 +66,7 @@ PostgresDestination({
       targetType: "varchar",
     },
   ],
-})
+});
```
Comment (GodsonLeigh) [inline src/destinations/README.md:69]: does this need the ; as our JS standard does not typically allow those to be used without flagging an issue.

File: src/destinations/README.md [hunk 2]
```diff
@@ -98,9 +99,9 @@ SQLFileDestination({
...
```
Comment (GodsonLeigh) [inline src/destinations/README.md:102]: same as above

**Review by GodsonLeigh — COMMENTED** [review]
couple of queries

File: src/destinations/README.md (hunk 1 above)
Comment (suityou01) [reply src/destinations/README.md:69]: I nuked it from orbit just to be sure

File: src/destinations/README.md (hunk 2 above)
Comment (suityou01) [reply src/destinations/README.md:102]: same as above ;-)

**Comment by suityou01** [comment]
Suspect sonarcloud has a bug that means it will not pick up the newly created sonar-project.properties file at the root of the project and will continue to run analysis on the tests. Closing this PR and creating another one in the hope that playing this hunch gets over this.

**Pull request closed without merging**